pronotepy/dataClasses.py

```diff
@@ -123,6 +123,12 @@ def date_parse(formatted_date: str) -> datetime.date:
         elif re.match(r"\d{2}/\d{2}", formatted_date):
             formatted_date += f"/{datetime.date.today().year}"
             return datetime.datetime.strptime(formatted_date, "%d/%m/%Y").date()
+        elif re.match(r"\d{2}\d{2}$", formatted_date):
+            date = datetime.date.today()
+            hours = int(formatted_date[:2])
+            minutes = int(formatted_date[2:])
+            formatted_date = f"{date:%d}/{date:%m}/{date.year} {hours}h{minutes}"
```

Comment:
Are you sure this is correct? `hours` and `minutes` are `int`s, so by default, there are no leading zeroes. AFAIK `%Hh%M` only matches when it has leading zeroes.

Reply:
Never mind, tried it, and it works.